File: app/src/main/java/com/thedancercodes/androidcv345/DisplayImage.java
package com.thedancercodes.androidcv345;

import android.graphics.Bitmap;
import android.graphics.BitmapFactory;
import android.os.Bundle;
import android.widget.ImageView;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

public class DisplayImage extends AppCompatActivity {

    ImageView imageView;

    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_display_image);
        imageView = findViewById(R.id.displayImageView);

        Bitmap bitmap = BitmapFactory.decodeFile(getIntent().getStringExtra("image_path"));
        imageView.setImageBitmap(bitmap);
    }
}

File: app/src/main/java/com/thedancercodes/androidcv345/DisplayImage.kt
package com.thedancercodes.androidcv345

import android.graphics.BitmapFactory
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_display_image.*
import kotlinx.android.synthetic.main.camera_layout.*

class DisplayImage : AppCompatActivity() {
    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_display_image)
        val bitmap = BitmapFactory.decodeFile(intent.getStringExtra("image_path"))
        displayImageView.setImageBitmap(bitmap)
    }
}